frankDoc/src/main/java/nl/nn/adapterframework/frankdoc/DocWriterNew.java

```diff
@@ -584,6 +639,9 @@ public void addDeclaredGroup() {
 				XmlBuilder group = createGroup(groupName);
 				xsdComplexItems.add(group);
 				XmlBuilder sequence = addSequence(group);
+				// This comes down to adding config child <Module> to the root element <Configuration>.
+				// The name Module comes from a constant declaration in Constants.
+				referencedEntityRootAdder.accept(sequence);
```

Comment:
Waar krijgt deze referencedEntityRootAdder ergens een 'nuttige' waarde?

Reply:
De aanpak was inderdaad nodeloos ingewikkeld. Ik geef nu geen Consumer meer door die bijna altijd leeg is. Ik heb nu gewoon een if gemaakt.